### DISCUSSION TITLE
Primary left & right flight displays lost after night 26th/27th Jan. update "Unstable Build (with Custom FBW 58af13b)"

### DISCUSSION THREAD

**Discussion by Mickepilot** [flybywiresim/aircraft#3206] [Q&A]
Anybody how can help me?
After last _A32NX Unstable Master Branch Build (with Custom FBW 58af13b)_ update (night 26th/27th Jan.) the **left & right primary flight displays** does not work anymore (Sim set-up with flight started **airborn** during final approach into LSZH)
The displays only show "SPD, ATT, ALT V/S" in blinking red colour. Everything else works fine as before, also STBY (horizon) instrument. No other warnings, also Overhead Panel seems ok.
I did try with removing the A32NX from the community file, using the regular Asobo A320neo, then the instruments where ok.
But once I put the A32NX file back, the above problem came back.

After futher testing I found that starting a flight **airborne** into EDDF ILS 25C (Frankfurt) also shows L & R primary displays u/s.
But when **starting a flight on ground** (RWY 28 LSZH) all flight instruments are working.
Funny. like some kind of initialization missing when airborne, that's new.

**Comment by ExampleWasTaken** [comment]
The custom FBW is experimental and not supported. At least for now.

**Reply by Mickepilot** [reply]
@ExampleWasTaken, thank's for your feedback. I'm aware of the non support of the experimental version, that's the reason I asked about the flt. display. issue under "Discussions".
The reason I use the experimental "Unstable Build (with Custom FBWversion)", it is the nice support of the Thrustmaster Airbus Edition TCA Quadrant.

**Comment by tracernz** [comment]
> Funny. like some kind of initialization missing when airborne, that's new.

Precisely that. Apologies. The fix was opened yesterday and shouldn't be too far away https://github.com/flybywiresim/a32nx/pull/3223

**Reply by Mickepilot** [reply]
@tracernz, excellent fix, both primary flt. displays working great again. Thank you for your help.

**Reply by Riggie** [reply]
I have the " Stable version " with exactly the same problem, on a cold start it happened and they never booted up, I flew for at least 20 mins but they didn't appear.  Also when starting an Activity like Paro it happened once,  all panels were black with the red fault writing on them but then they slowly booted up?  I took a screenshot of the fault lights on the overhead panel section GPWS, sure they have something to do with it,  tried operating all those switches but they did not help. If I start on the Runway it doesn't seem to happen at all?  I don't know much about the Sim yet, but do these 4 panels connect/rely on online data?
 
![GPWS](https://user-images.githubusercontent.com/77942251/106491590-edb3c200-64ae-11eb-8131-f04f7865c029.jpg)

**Reply by Mickepilot** [reply]
@Riggie, as mentioned above I use the "Unstable Build (with Custom FBWversion)" because of the Thrustmaster Airbus Ed. TCA Quadrant. I just started the sim. again, direct airborne, on final app. in LSZH, RWY 14, everything works fine for me. You must have some other problem.

**Reply by Riggie** [reply]
Well if anything, it's good to know a similar problem has been sorted in the later version, I just avoid cold starts and everything is fine...................... I also have the Airbus TCA Quadrant and I love it,  the control is so good.
